api/design/category.go

```diff
@@ -26,12 +27,11 @@ var _ = Service("category", func() {
 	Method("list", func() {
 		Description("List all categories along with their tags sorted by name")
 		Result(func() {
-			Attribute("data", ArrayOf(Category))
+			Attribute("data", ArrayOf(types.Category))
 		})
 
 		HTTP(func() {
 			GET("/categories")
-			GET("/v1/categories")
```

Comment:
This will be breaking change now as we already published

Reply:
added back.. API will be removed in the next release and be internal to hub